spec/controllers/bids_controller_spec.rb

```diff
@@ -67,6 +69,39 @@
     end
   end
 
+  describe '#index' do
+    let(:bids) {
+      [3400, 3000, 1800].map do |amount|
+        Bid.create({
+          auction_id: auction.id,
+          amount: amount,
+          bidder_id: current_bidder.id
+        })
+      end
+    }
+    context 'when logged in' do
+      before do
+        allow(controller).to receive(:current_user).and_return(current_bidder)
+      end
+
+      it 'renders the template' do
+        get :index, auction_id: auction.id
+        expect(response).to render_template(:index)
+      end
+    end
+
+    context 'when logged out' do
+      before do
+        allow(controller).to receive(:current_user).and_return(nil)
+      end
+
+      it 'renders the template' do
```

Comment:
what do you think about not including a test for this? Since this controller is using Rails default template, it seems like this is testing Rails internals more than anything else